Mathlib/FieldTheory/Normal.lean

```diff
@@ -474,89 +474,3 @@ theorem isSolvable_of_isScalarTower [Normal F K₁] [h1 : IsSolvable (K₁ ≃
 #align is_solvable_of_is_scalar_tower isSolvable_of_isScalarTower
 
 end lift
-
-section normalClosure
```

Comment:
Can you make a quick PR that moves this content with no other changes, to make the diff easier to folllow?

Reply:
Sure, will do.